File: branches/V1.0/proj/WinStore_FamilyAccounting/WinStore_FamilyAccountingCore/Exceptions/NotInValueRangeException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace WinStore_FamilyAccountingCore.Exceptions
{
    public class NotInValueRangeException : Exception
    {
        public NotInValueRangeException(String fieldName, Object obj)
            : base(String.Format("Value {0} is not in value range {1}.",
                    obj.ToString(), fieldName))
        {
        }
    }
}

File: branches/V1.0/proj/CoreLibrary/src/com/jasonzqshen/familyaccounting/core/exception/NotInValueRangeException.java
package com.jasonzqshen.familyaccounting.core.exception;

public class NotInValueRangeException extends Exception {
	/**
	 * 
	 */
	private static final long serialVersionUID = -7365572226472484978L;

	public NotInValueRangeException(String fieldName, Object object) {
		super(String.format("Value %s is not in value range %s.",
				object.toString(), fieldName));
	}
}
